src/NLog.Extensions.Logging/Logging/NLogProviderOptions.cs

```diff
@@ -56,6 +56,18 @@ public class NLogProviderOptions
 #endif
         public bool IncludeActivtyIdsWithBeginScope { get; set; }
 
+        /// <summary>
+        /// Reset the default Microsoft Logger Filter Options for the <see cref="NLogLoggerProvider"/>
+        /// </summary>
+        /// <remarks>This option affects the building of service configuration, so assigning it from appsettings.json has no effect (loaded after).</remarks>
+        public bool ResetLoggerProviderFilter { get; set; }
```

Comment:
Is a reset, or maybe a remove? 

Maybe RemoveDefaultFilters?

I don't know if users know what a LoggerProvider is

Reply:
NLog.Extensions.Logging is a LoggerProvider for MEL-LoggerFactory. Anyway it will change to default = true with ver. 5.0

Reply:
Instead of `ResetLoggerProviderFilter` then it could also `RemoveLoggerFactoryFilter` ?